.github/workflows/android-prod-deploy.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     tags:
       - android-*
+      - '[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
very rarely have we added a `-X` suffix to the version number to re-deploy a tag, so added the extra `*` just in case.